ui/page/root/wallet_selector_page.go

```diff
@@ -27,6 +30,11 @@ type badWalletListItem struct {
 	deleteBtn cryptomaterial.Button
 }
 
+type walletIndexTuple struct {
+	AssetType  libutils.AssetType
+	LocalIndex int
+}
```

Comment:
This here is what we need. we dont need it redeclared in the wallet selector widget. Thats where the complexity came in.